app/src/main/java/org/schabi/newpipe/MainActivity.java

```diff
@@ -139,8 +139,8 @@ protected void onCreate(final Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
 
-        if (getSupportFragmentManager() != null
-                && getSupportFragmentManager().getBackStackEntryCount() == 0) {
+        getSupportFragmentManager();
+        if (getSupportFragmentManager().getBackStackEntryCount() == 0) {
```

Comment:
Please explain this change.

Reply:
`getSupportFragmentManager` is never null (annotated as such), so it can be simplified to above. Though I now realize that [this](https://github.com/TeamNewPipe/NewPipe/pull/4929/files/04cb6ba3d0d575a3440ff4ef6fbade3cd9013830..31fead4e873e915d7c327e4c2eff7c550554856e#diff-cfbf289c5984216024c944c30dce7cffebc87a38816e3c3f5aa3d266b65e6afcR142) was an oops, idk why I did that. I'll fix it.

Reply:
Thanks.